app/controllers/review_controller.rb

```diff
@@ -0,0 +1,58 @@
+class ReviewController < ApplicationController
```

Comment:
To properly follow Rails convention, this should be called `reviews_contoller` (plural reviews) - plural in both the file name and the class name